example/geometry/t8_example_geometries.cxx

```diff
@@ -511,6 +513,23 @@ t8_analytic_geom (int level, t8_example_geom_type geom_type)
       snprintf (vtuname, BUFSIZ, "forest_moebius_lvl_%i", level);
     }
     break;
+  case T8_GEOM_TWO_GEOMETRIES:
+    t8_global_productionf
+      ("Creating uniform level %i forest with a cylinder and a sinus cosinus geometry.\n",
+       level);
+    /* Cylinder geometry on tree 0. Sincos geometry on tree 1. */
+    geometry = new t8_geometry_cylinder;
+    geometry_sincos = new t8_geometry_sincos;
+    t8_cmesh_set_tree_class (cmesh, 0, T8_ECLASS_QUAD);
+    /* Tree 0 is connecte to itself to form a cylinder */
+    t8_cmesh_set_join (cmesh, 0, 0, 0, 1, 0);
+    t8_cmesh_set_tree_class (cmesh, 1, T8_ECLASS_QUAD);
+    t8_cmesh_register_geometry (cmesh, geometry_sincos);
```

Comment:
We should comment, that both geometries have to be registered and that the cylinder geometry is registered later on

Reply:
done